app/addons/config/assets/less/config.less

```diff
@@ -9,31 +9,55 @@
 //  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
 //  License for the specific language governing permissions and limitations under
 //  the License.
-//
+
 @import "../../../../../assets/less/variables.less";
 @import "../../../../../assets/less/bootstrap/mixins.less";
 
-
-
 .config-item {
   height: 65px;
 
-  .js-value-input {
+  .config-value-form {
+    button {
+      margin-left: 8px;
+    }
+  }
+
+  .config-value-input {
     width: 80%;
     margin: 0;
   }
-  .js-show-value,
-  .js-delete-value {
+
+  .config-show-value,
+  .config-delete-value {
     cursor: pointer;
   }
-  .js-hidden {
-    display: none;
-  }
+
   .text-center {
     text-align: center;
   }
 
   button { width: 7%;}
+
+  transition: background-color 100ms;
+}
+
+.config-item:hover {
+  background-color: #e0e0e0;
+}
+
+.table-striped tbody > tr.config-item:nth-child(odd) > td,
+.table-striped tbody > tr.config-item:nth-child(odd) > th {
+  transition: background-color 100ms;
+}
+
+.table-striped tbody > tr.config-item:nth-child(odd):hover > td,
+.table-striped tbody > tr.config-item:nth-child(odd):hover > th {
+  background-color: #e7e7e7;
+}
```

Comment:
this changes the color for all striped tables in fauxton - is that the intention?